src/nasuta-sebastian/Fsd.Sebastian.Cs/Fsd.Sebastian.Cs1/CharCounter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Fsd.Sebastian.Cs1
+{
+    class CharCounter
+    {
+        public void Run(string[] args)
+        {
+            Console.WriteLine("Console arguments:\n");
+            if (args.Length == 0) Console.WriteLine("No arguments\n");
+            else for (int i = 0; i < args.Length; i++)
```

Comment:
wygląda to jakby było wyrażenie "else for" zrób nową linię oraz dodaj klamry do else { } aby było widać, że for to pierwsza instrukcja w bloku else. Równie dobrze można zrobić metodę, która uporządkuje kod